src/templates/use-cases.tsx

```diff
@@ -25,135 +39,66 @@ import Logo from "../components/Logo"
 import MeetupList from "../components/MeetupList"
 import PageMetadata from "../components/PageMetadata"
 import RandomAppList from "../components/RandomAppList"
-import UpgradeTableOfContents, {
-  Item as ItemTableOfContents,
-} from "../components/UpgradeTableOfContents"
-import TableOfContents, { Item } from "../components/TableOfContents"
+
+import UpgradeTableOfContents from "../components/UpgradeTableOfContents"
+import TableOfContents from "../components/TableOfContents"
 import Translation from "../components/Translation"
 import SectionNav from "../components/SectionNav"
 import {
   Divider,
   Paragraph,
-  Header1,
-  Header4,
   ListItem,
```

Comment:
We need to deprecate these components as well.

- Paragraph => `Text`
- ListItem => `<chakra.li>`